howto/regex.po

```diff
@@ -32,160 +32,250 @@ msgstr ""
 #: ../Doc/howto/regex.rst:18
 msgid ""
 "This document is an introductory tutorial to using regular expressions in "
-"Python with the :mod:`re` module.  It provides a gentler introduction than "
-"the corresponding section in the Library Reference."
+"Python with the :mod:`re` module.  It provides a gentler introduction than the "
```

Comment:
Please configure your po editor to avoid changing msgids. Using "poindent" from pypi may help sometimes reformatting your po file before submitting them. having diff in msgids make git diffs hard to read.